src/vs/platform/actions/browser/buttonbar.ts

```diff
@@ -99,9 +100,15 @@ export class WorkbenchButtonBar extends ButtonBar {
 				});
 			} else {
 				action = actionOrSubmenu;
+				const kb = this._keybindingService.lookupKeybinding(action.id);
+				if (kb) {
+					tooltip = localize('labelWithKeybinding', "{0} ({1})", action.tooltip || action.label, kb.getLabel());
+				} else {
+					tooltip = action.tooltip || action.label;
+				}
 				btn = this.addButton({
 					secondary: conifgProvider(action, i)?.isSecondary ?? secondary,
-					ariaLabel: action.label,
+					ariaLabel: tooltip,
```

Comment:
Does this need to be fixed in the `addButtonWithDropdown` case above too?

Reply:
That doesn't support an aria label property